internal/server/kong/ws/config/version_compatibility.go

```diff
@@ -263,6 +263,42 @@ func (vc *WSVersionCompatibility) processConfigTableUpdates(uncompressedPayload
 	return processedPayload, nil
 }
 
+func (vc *WSVersionCompatibility) removeCoreEntity(
+	processedPayload string,
+	entityType string,
+	dataPlaneVersionStr string,
+	configTableKey string,
+	changeID ChangeID,
+	tracker *ChangeTracker,
+) string {
+	entityIDs := gjson.Get(processedPayload, fmt.Sprintf("config_table.%s.#.id", configTableKey))
+	for _, id := range entityIDs.Array() {
+		err := tracker.TrackForResource(changeID, ResourceInfo{
```

Comment:
Please update this to use Track() instead. When a change affects *all* resources, global tracking must be used. This is to ensure that the user fixes the problem at a global level rather than trying to figure out the problem within the context of a resource (which could mislead the user).